lib/src/geo/crs.dart

```diff
@@ -417,7 +417,11 @@ abstract class Projection {
   /// longitudes -179 and 179 to be projected each on one side.
   /// [referencePoint] is used for polygon holes: we want the holes to be
   /// displayed close to the polygon, not on the other side of the world.
-  List<Offset> projectList(List<LatLng> points, {LatLng? referencePoint}) {
+  List<Offset> projectList(
+    List<LatLng> points, {
+    LatLng? referencePoint,
+    required bool oneWorld,
```

Comment:
@monsieurtanuki I was referring to this line. Can we default it to `false`?

I think the changes to offsets.dart were OK because it wasn't exported. Maybe it's easier to revert that last commit?

Reply:
@JaffaKetchup Done.